Ethane

```diff
@@ -0,0 +1,4 @@
+methyl = [0,2,3,4]
+list = [methyl, methyl]
+
+guess_driver("Ethane", list, 1)
```

Comment:
it should be `"Ethane.sdf"` also the number of unpaired electrons needs to be per fragment `[1, 1]`